libs/cart/state/src/effects/cart-resolver.effects.ts

```diff
@@ -36,6 +43,11 @@ export class DaffCartResolverEffects<T extends DaffCart = DaffCart> {
       map(resp => new DaffResolveCartSuccess(resp)),
       catchError(error => {
         switch(true) {
+          case error instanceof DaffServerSideStorageError:
+            return this.store.pipe(
+              select(getDaffCartSelectors<T, V, U>().selectCartValue),
+              map(cart => new DaffResolveCartSuccess(cart))
+            );
```

Comment:
So this switch case would only ever be triggered server side, right? 

How do we know that it should always resolve with success if no cart was retrieved? Wouldn't the cart in state always be null at this point, because it couldn't even get the id from local storage? Doesn't this mean that we should never use the resolve cart guard server side, because the server can never verify that there is a valid cart id in local storage? Should the ResolveCartGuard just always resolve immediately server-side?